src/components/cards/CollaborationsCard.jsx

```diff
@@ -11,8 +11,10 @@ import DataDisplay from '../dataDisplays/DataDisplay';
 import CollaborationsDialog from './collaborations/CollaborationsDialog';
 
 export default function CollaborationsCard({
+  title,
   userId,
   htmlId = null,
+  noCollaborationsMsg,
```

Comment:
Do `title` and `noCollaborationsMsg` need to be props? They never change.

Reply:
A lot of these things are vestiges of when I was populating a normal collaboration table with a userId that was different form the current viewer. 

I kept a few of the things that I thought generalized the code in ways that might future-proof the product.

But you're right; I shouldn't plan for a use case we demonstrably no longer have.